docs/components/movement-sensor/gps/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Configure a GPS"
+linkTitle: "GPS"
+weight: 2
+type: "docs"
+description: "Supported GPS models."

```

Comment:
```suggestion
description: "Supported GPS models."
empty_node: true
```
this will make the page just a toggle on the side bar. 

I'd suggest moving the paragraph onto each page. That's then fewer clicks for the user and they have the supported methods right there (i'd suggest putting them in a list) and linking each to the main page.

Reply:
Okay will do the same for IMU too